package.json

```diff
@@ -18,31 +18,33 @@
     "postinstall": "node node_modules/grunt-cli/bin/grunt install"
   },
   "dependencies": {
+    "async": "~0.2.9",
+    "bcryptjs": "~1.0.4",
+    "bower": "~1.3.8",
+    "connect-flash": "~0.1.1",
+    "connect-mongo": "~0.3.3",
+    "console-stamp": "~0.1.1",
+    "dotenv": "^2.0.0",
```

Comment:
Oh ok!
Didn't see that! #MyBad

But seeing there's no `.env` file what happens?